getting-started/installation-windows.rst

```diff
@@ -90,18 +97,31 @@ Install EPICS
 
 There should be lots of warnings, but no error. 
 
-Test EPICS
-----------
+Check EPICS Installation
+------------------------
 
 Run ``softIoc`` and, if everything is ok, you should see an EPICS prompt.
 
 ::
 
-    softIoc
-    epics>
+    $ /home/Hitesh/base-7.0.3.1/bin/windows-x64-mingw/softIoc
+    $ epics>
 
 You can exit with ctrl-c or by typing exit.
 
 Voilà.
 
 Ok, now you know that EPICS is installed correctly.
+
+Test EPICS in Windows
+---------------------
+
+Exit or minimise Msys2 environment. Open windows command prompt.
+
+::
+
+     cd c:\msys64\home\Hitesh\base-7.0.3.1\bin\windows-x64-mingw
+     softIoc.exe
+     dbLoadDatabase("c:\msys64\home\Hitesh\base-7.0.3.1\bin\windows-x64-mingw\../../dbd/softIoc.dbd")
+     softIoc_registerRecordDeviceDriver(pdbbase)
```

Comment:
There's an easier way to test the softIoc without having to type the iocsh commands. The `cd` command isn't really necessary here either since the executable will look in its own bin directory for the DLLs that it needs. This should be all you need to type:

```
C:\>\epics\base-7.0.3.1\bin\windows-x64-mingw\softIoc -x test
Starting iocInit
############################################################################
## EPICS R7.0.3.1
## EPICS Base built Feb 27 2020
############################################################################
iocRun: All initialization complete
epics> dbl
test:BaseVersion
test:exit
epics> exit

C:\>
```

That runs the softIoc, loads 2 records from a built-in database file and then runs iocInit. The parameter to the `-x` flag is a prefix for the records. I also added the `dbl` command at the iocsh prompt to show that those records have been loaded, and ran exit when I was finished.

Reply:
I tried to match "Installation on Linux/UNIX/DARWIN" procedures. So that documentation looks uniform. But I can change to your suggestion, it looks simpler.

Reply:
Updated as advised. Committed.